pkg/cmos/cmos.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2012-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package cmos lets you read and write to cmos registers while doing basic checks on valid register selections.
+
+package cmos
+
+import (
+	"fmt"
+	"github.com/u-root/u-root/pkg/memio"
+)
+
+const (
+	CMOS_REG_PORT  = 0x70
+	CMOS_DATA_PORT = 0x71
+)
+
+func Read(reg int64, data memio.UintN) error {
```

Comment:
Leave a comment on every exported symbol that starts with the symbol name. Should consist of full sentences: https://golang.org/doc/effective_go.html#commentary

These get parsed and put into godocs. (No, our code base is terrible at doing this consistently; we're trying to apply a consistent standard to new code reviews, though. This is equivalent to internal Go readability.)

```go
// Read reads a register reg from CMOS into data.
func Read...
```